### DISCUSSION TITLE
Prompt a Date-selection box when giving the title for new note

### DISCUSSION THREAD

**Discussion by selinaZitrone** [chhoumann/quickadd#558] [Q&A]
Hi everyone,
I try to create a QuickAdd action but I fail to achieve my goal, maybe someone can help me.

What I want to do:
I have a template for meetings and when I create a new meeting note from the template, I want the following:

- Filename format: `YYYY-MM-DD name of meeting` with the date being the date of the meeting some time in the future.
- The meeting date from the file title should be included as a field in the YAML frontmatter.
- Ideally the meeting date can be selected from a popup calendar when creating the note

What I did so far:
I created a QuickAdd action that creates the file from template. I used ` {{VDATE:<variable name>, <date format>}}` together with the natural language dates plugin to include the meeting date in the note title. This works but there are still two problems:

1) The date selection isn't through a popup calendar, and most importantly:
2) I can't use it as variable in the frontmatter.

Do you see any easy solution to my problem?

Thank you in advance for the help ^_^

**Comment by chhoumann** [comment]
Very late answer - sorry about that. Both of your remaining problems are solved on current QuickAdd.

**2) Reusing the date in frontmatter:** `{{VDATE}}` variables work like any other variable - enter once, use anywhere. Put the same variable name in both places and one prompt fills both:

File Name Format:

```
{{VDATE:meeting,YYYY-MM-DD}} {{VALUE:name of meeting}}
```

Template frontmatter:

```yaml
date: {{VDATE:meeting,YYYY-MM-DD}}
```

You can even reuse `meeting` with a different date format elsewhere in the note - it's one prompt, formatted per token.

**1) Popup calendar:** the `{{VDATE}}` prompt has opened a native calendar picker since QuickAdd 2.10.0, no extra plugin needed. Natural language input ("next tuesday") still works alongside it, and `|time` adds an HH:mm control if you ever need it.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#vdate

Closing as resolved.
